buffer/src/test/java/io/netty5/buffer/api/tests/SensitiveBufferTest.java

```diff
@@ -106,6 +108,30 @@ public void closingBuffersWithComplicatedStructuralSharingMustNotZeroBytesUntilA
         }
     }
 
+    @Test
+    public void sensitiveAllocatorMustCaptureMemoryManager() {
+        MemoryManager baseMemoryManager = MemoryManager.instance();
+        StubManager stubManager = new StubManager(baseMemoryManager);
+
+        {
+            BufferAllocator allocator = MemoryManager.using(stubManager, () -> sensitiveOffHeapAllocator());
```

Comment:
Shouldn’t we close the allocator once done (consider using try with resources) ? Same comment everywhere

Reply:
It is now